score-lang-cli/src/main/java/org/openscore/lang/cli/services/ScoreServicesImpl.java

```diff
@@ -131,6 +131,37 @@ public synchronized void onEvent(ScoreEvent scoreEvent) throws InterruptedExcept
                         printWithColor(Ansi.Color.YELLOW, prefix + taskName);
                     }
                     break;
+                case ScoreLangConstants.EVENT_OUTPUT_END:
```

Comment:
all the logic must be in a diff method..
moreover.. i do not understand what you are doing here, why so much parsing is needed?